### PR TITLE
CORCI-1176 build: Hold back mock-core-configs (#8216)

### PR DESCRIPTION
**Pull request by jolivier23**
*As the update replaces epel-8-* configs with <distro>+epel-8-* which
we are not ready for yet.
* Use Docker CE runner

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

### CODE REVIEW

**Review by phender — APPROVED** [review]

**Review by brianjmurrell — DISMISSED** [review]

File: Jenkinsfile
Comment (ashleypittman) [inline Jenkinsfile:316]: This part of the change is a hack for the docker builders which shouldn't be needed after todays shutdown and will need to be backed out of master, so for 2.0 can we wait until later and land this PR without this component of the change.

**Review by brianjmurrell — APPROVED** [review]

**Pull request merged**
